### PR TITLE
Added Travis File

### PR DESCRIPTION
**Pull request by sriramnjr7**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Certificate Generator Server
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/06bd6acff9cd4506985192596642ef5f)](https://www.codacy.com/app/JBossOutreach/certificate-generator-server?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=JBossOutreach/certificate-generator-server&amp;utm_campaign=Badge_Grade)
+![Android](https://img.shields.io/badge/GCI-Android-green.svg?longCache=true&style=flat-square)   ![In Progress](https://img.shields.io/badge/In--progress-true-green.svg?longCache=true&style=flat-square) <br />
```
Comment (divyanshkul) [inline README.md]: @sriramnjr7  this is not an android project, you should remove this.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
...
+[![Build Status](https://travis-ci.org/JbossOutreach/visiting-card-android.svg?branch=master)](https://travis-ci.org/JbossOutreach/visiting-card-android)
```
Comment (divyanshkul) [inline README.md]: This is not the ```visiting-card-android``` repository. 
Please don't just copy paste stuff from other repos :))
Also, please change it to "JBossOutreach" at both the places.

**Comment by divyanshkul** [comment]
@aashutoshrathi please verify the travis config on this.

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,47 @@
+language: python
+matrix:
+  include:
+    - python: 2.7
+    - python: 3.4
```
Comment (aashutoshrathi) [inline .travis.yml]: I think we can remove 3.4, as it doesn't create different issues than 3.5
and we are not creating CI for a python package rather for a server.

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    - python: 3.5
+    - python: 3.6
+    - python: 3.7
+      dist: xenial
+      sudo: true
+    - python: pypy2.7-5.10.0
+    - python: pypy3.5-5.10.1
+  allow_failures:
+    - python: pypy2.7-5.10.0
```
Comment (aashutoshrathi) [inline .travis.yml]: Also, remove pypy2.7

**Comment by prateekiiest** [comment]
I agree with @aashutoshrathi. Else LGTM

**Comment by aashutoshrathi** [comment]
Ping @sriramnjr7

**Pull request closed without merging**